app/services/response_file_builder/build_response_pdf_file.rb

```diff
@@ -1,14 +1,16 @@
 module ResponseFileBuilder
-  class BuildResponsePdfFile
+  class BuildResponsePdfFile # rubocop:disable Metrics/ClassLength
```

Comment:
This class has now changed significantly.  Instead of hard coding the field names, it now looks them up from a yaml file named the same as the pdf file.
This allows us to have multiple pdf templates where the fields are normalized so that the code doesnt need to change for each template.